todo.sh

```diff
@@ -1263,8 +1264,8 @@ case $action in
 "listfile" | "lf" )
     shift  ## Was listfile, next $1 is file name
     if [ $# -eq 0 ]; then
-        [ $TODOTXT_VERBOSE -gt 0 ] && echo "Files in the todo.txt directory:"
-        cd "$TODO_DIR" && ls -1 *.txt
+        [ "$TODOTXT_VERBOSE" -gt 0 ] && echo "Files in the todo.txt directory:"
+        cd "$TODO_DIR" && ls -1 -- *.txt
```

Comment:
Can you explain this change?

Reply:
This fixes `SC2035: Use ./*glob* or -- *glob* so names with dashes won't become options.`

The '--' in ls stops option processing, so that file names beginning with a hyphen are not treated as options.